utils/build-script

```diff
@@ -2018,6 +2027,11 @@ iterations with -O",
     parser.add_argument(
         "--enable-tsan-runtime",
         help="enable Thread Sanitizer on the swift runtime")
+    parser.add_argument(
+        "--enable-lsan",
+        help="enable Leaks Sanitizer for swift tools",
```

Comment:
I have a few other things to take care of today, so I am going to loop back around.